redfish-core/include/registries/openbmc.json

```diff
@@ -1687,6 +1687,15 @@
             "ParamTypes": ["string"],
             "Resolution": "None.",
             "Severity": "Critical"
+        },
+        "HardwareIsolationReason": {
```

Comment:
Shouldn't the version of the registry file be updated since there is a new message added?

I also thought they were supposed to be added in alphabetical order, so seems like this should be moved up in the file.

Reply:
Do we need to really put it in alphabetical order here?